Crash on import of certain mp3 files

Preface: I'm a DJ and have an extensive audio library that is pretty well curated. Certain tracks crash airtime_analyzer when importing, with this error:

```
2018-11-26 14:26:19,798 [message_listener] [INFO ]   - Received '{"tmp_file_path":"\/srv\/airtime\/stor\/organize\/TLC - Creep (BPM Supreme ReDrum) (Clean Short Edit).mp3","storage_backend":"file","import_directory":"\/srv\/airtime\/stor\/\/imported\/1","original_filename":"TLC - Creep (BPM Supreme ReDrum) (Clean Short Edit).mp3","callback_url":"http:\/\/35.192.238.53\/\/rest\/media\/25","api_key":"2PA4WDCTHECXHQR1FSML","file_prefix":""}' on routing_key ''
2018-11-26 14:26:19,814 [analyzer_pipeline] [ERROR]  u'replaygain_seratogain_gain'
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/airtime_analyzer-0.1-py2.7.egg/airtime_analyzer/analyzer_pipeline.py", line 66, in run_analysis
    metadata = MetadataAnalyzer.analyze(audio_file_path, metadata)
  File "/usr/local/lib/python2.7/dist-packages/airtime_analyzer-0.1-py2.7.egg/airtime_analyzer/metadata_analyzer.py", line 57, in analyze
    if audio_file == None:  # Don't use "if not" here. It is wrong due to mutagen's design.
  File "/usr/lib/python2.7/dist-packages/mutagen/_util.py", line 192, in __eq__
    return dict(self.items()) == other
  File "/usr/lib/python2.7/dist-packages/mutagen/_util.py", line 134, in items
    return list(izip(self.keys(), self.values()))
  File "/usr/lib/python2.7/dist-packages/mutagen/_util.py", line 128, in values
    return [self[k] for k in self.keys()]
  File "/usr/lib/python2.7/dist-packages/mutagen/_file.py", line 56, in __getitem__
    return self.tags[key]
  File "/usr/lib/python2.7/dist-packages/mutagen/easyid3.py", line 196, in __getitem__
    return func(self.__id3, key)
  File "/usr/lib/python2.7/dist-packages/mutagen/easyid3.py", line 385, in gain_get
    raise EasyID3KeyError(key)
EasyID3KeyError: u'replaygain_seratogain_gain'
2018-11-26 14:26:19,817 [message_listener] [ERROR]  Analyzer pipeline exception: u'replaygain_seratogain_gain'
2018-11-26 14:26:19,857 [connectionpool] [INFO ]  Starting new HTTP connection (1): 35.192.238.53
2018-11-26 14:26:20,004 [status_reporter] [INFO ]  HTTP request sent successfully.
```

The error seems to point to mutagen, but running:
```
mutagen-inspect /srv/airtime/stor/organize/TLC\ -\ Creep\ \(BPM\ Supreme\ ReDrum\)\ \(Clean\ Short\ Edit\).mp3 
-- /srv/airtime/stor/organize/TLC - Creep (BPM Supreme ReDrum) (Clean Short Edit).mp3 
- MPEG 1 layer 3, 320000 bps (CBR?), 44100 Hz, 2 chn, 123.90 seconds (audio/mp3)
APIC= (image/jpeg, 6834 bytes)
GEOB=[unrepresentable data]
GEOB=[unrepresentable data]
GEOB=[unrepresentable data]
GEOB=[unrepresentable data]
GEOB=[unrepresentable data]
GEOB=[unrepresentable data]
GEOB=[unrepresentable data]
RVA2=[unrepresentable data]
TBPM=93
TCON=Hip Hop
TDRC=2018
TIT2=Creep (BPM Supreme ReDrum) (Clean Short Edit)
TKEY=Fm
TPE1=TLC 
```
Yields good results and no crash.

--- a/python_apps/airtime_analyzer/setup.py
+++ b/python_apps/airtime_analyzer/setup.py
@@ -28,7 +28,7 @@ setup(name='airtime_analyzer',
       packages=['airtime_analyzer'],
       scripts=['bin/airtime_analyzer'],
       install_requires=[
-          'mutagen==1.31', # The Mutagen guys change stuff all the time that break our unit tests. Watch out for this.
+          'mutagen', # got rid of specific version requirement 
           'pika',
           'daemon',
           'file-magic',

--- a/python_apps/airtime_analyzer/tests/metadata_analyzer_tests.py
+++ b/python_apps/airtime_analyzer/tests/metadata_analyzer_tests.py
@@ -24,8 +24,7 @@ def test_mp3_mono():
     metadata = MetadataAnalyzer.analyze(u'tests/test_data/44100Hz-16bit-mono.mp3', dict())
     check_default_metadata(metadata)
     assert metadata['channels'] == 1
-    assert metadata['bit_rate'] > 63000
-    assert metadata['bit_rate'] < 65000
+    assert metadata['bit_rate'] == 63998
     assert abs(metadata['length_seconds'] - 3.9) < 0.1
     assert metadata['mime'] == 'audio/mp3' # Not unicode because MIMEs aren't.
     assert metadata['track_total'] == u'10' # MP3s can have a track_total
@@ -35,8 +34,7 @@ def test_mp3_jointstereo():
     metadata = MetadataAnalyzer.analyze(u'tests/test_data/44100Hz-16bit-jointstereo.mp3', dict())
     check_default_metadata(metadata)
     assert metadata['channels'] == 2
-    assert metadata['bit_rate'] < 130000
-    assert metadata['bit_rate'] > 127000
+    assert metadata['bit_rate'] == 127998
     assert abs(metadata['length_seconds'] - 3.9) < 0.1
     assert metadata['mime'] == 'audio/mp3'
     assert metadata['track_total'] == u'10' # MP3s can have a track_total
@@ -45,8 +43,7 @@ def test_mp3_simplestereo():
     metadata = MetadataAnalyzer.analyze(u'tests/test_data/44100Hz-16bit-simplestereo.mp3', dict())
     check_default_metadata(metadata)
     assert metadata['channels'] == 2
-    assert metadata['bit_rate'] < 130000
-    assert metadata['bit_rate'] > 127000
+    assert metadata['bit_rate'] == 127998
     assert abs(metadata['length_seconds'] - 3.9) < 0.1
     assert metadata['mime'] == 'audio/mp3'
     assert metadata['track_total'] == u'10' # MP3s can have a track_total
@@ -55,8 +52,7 @@ def test_mp3_dualmono():
     metadata = MetadataAnalyzer.analyze(u'tests/test_data/44100Hz-16bit-dualmono.mp3', dict())
     check_default_metadata(metadata)
     assert metadata['channels'] == 2
-    assert metadata['bit_rate'] < 130000
-    assert metadata['bit_rate'] > 127000
+    assert metadata['bit_rate'] == 127998
     assert abs(metadata['length_seconds'] - 3.9) < 0.1
     assert metadata['mime'] == 'audio/mp3'
     assert metadata['track_total'] == u'10' # MP3s can have a track_total
@@ -158,8 +154,7 @@ def test_mp3_bad_channels():
     metadata = MetadataAnalyzer.analyze(filename, dict())
     check_default_metadata(metadata)
     assert metadata['channels'] == 1
-    assert metadata['bit_rate'] < 65000
-    assert metadata['bit_rate'] > 63000
+    assert metadata['bit_rate'] < 63998
     assert abs(metadata['length_seconds'] - 3.9) < 0.1
     assert metadata['mime'] == 'audio/mp3' # Not unicode because MIMEs aren't.
     assert metadata['track_total'] == u'10' # MP3s can have a track_total

--- a/python_apps/airtime_analyzer/tests/metadata_analyzer_tests.py
+++ b/python_apps/airtime_analyzer/tests/metadata_analyzer_tests.py
@@ -154,7 +154,7 @@ def test_mp3_bad_channels():
     metadata = MetadataAnalyzer.analyze(filename, dict())
     check_default_metadata(metadata)
     assert metadata['channels'] == 1
-    assert metadata['bit_rate'] < 63998
+    assert metadata['bit_rate'] == 63998
     assert abs(metadata['length_seconds'] - 3.9) < 0.1
     assert metadata['mime'] == 'audio/mp3' # Not unicode because MIMEs aren't.
     assert metadata['track_total'] == u'10' # MP3s can have a track_total

--- a/python_apps/airtime_analyzer/tests/metadata_analyzer_tests.py
+++ b/python_apps/airtime_analyzer/tests/metadata_analyzer_tests.py
@@ -24,7 +24,8 @@ def test_mp3_mono():
     metadata = MetadataAnalyzer.analyze(u'tests/test_data/44100Hz-16bit-mono.mp3', dict())
     check_default_metadata(metadata)
     assert metadata['channels'] == 1
-    assert metadata['bit_rate'] == 64876
+    assert metadata['bit_rate'] > 63000
+    assert metadata['bit_rate'] < 65000
     assert abs(metadata['length_seconds'] - 3.9) < 0.1
     assert metadata['mime'] == 'audio/mp3' # Not unicode because MIMEs aren't.
     assert metadata['track_total'] == u'10' # MP3s can have a track_total
@@ -34,7 +35,8 @@ def test_mp3_jointstereo():
     metadata = MetadataAnalyzer.analyze(u'tests/test_data/44100Hz-16bit-jointstereo.mp3', dict())
     check_default_metadata(metadata)
     assert metadata['channels'] == 2
-    assert metadata['bit_rate'] == 129757
+    assert metadata['bit_rate'] < 130000
+    assert metadata['bit_rate'] > 127000
     assert abs(metadata['length_seconds'] - 3.9) < 0.1
     assert metadata['mime'] == 'audio/mp3'
     assert metadata['track_total'] == u'10' # MP3s can have a track_total
@@ -43,7 +45,8 @@ def test_mp3_simplestereo():
     metadata = MetadataAnalyzer.analyze(u'tests/test_data/44100Hz-16bit-simplestereo.mp3', dict())
     check_default_metadata(metadata)
     assert metadata['channels'] == 2
-    assert metadata['bit_rate'] == 129757
+    assert metadata['bit_rate'] < 130000
+    assert metadata['bit_rate'] > 127000
     assert abs(metadata['length_seconds'] - 3.9) < 0.1
     assert metadata['mime'] == 'audio/mp3'
     assert metadata['track_total'] == u'10' # MP3s can have a track_total
@@ -52,7 +55,8 @@ def test_mp3_dualmono():
     metadata = MetadataAnalyzer.analyze(u'tests/test_data/44100Hz-16bit-dualmono.mp3', dict())
     check_default_metadata(metadata)
     assert metadata['channels'] == 2
-    assert metadata['bit_rate'] == 129757
+    assert metadata['bit_rate'] < 130000
+    assert metadata['bit_rate'] > 127000
     assert abs(metadata['length_seconds'] - 3.9) < 0.1
     assert metadata['mime'] == 'audio/mp3'
     assert metadata['track_total'] == u'10' # MP3s can have a track_total
@@ -109,7 +113,8 @@ def test_mp3_utf8():
     assert metadata['genre'] == u'Я Б Г Д Ж Й'
     assert metadata['track_number'] == u'1'
     assert metadata['channels'] == 2
-    assert metadata['bit_rate'] == 129757
+    assert metadata['bit_rate'] < 130000
+    assert metadata['bit_rate'] > 127000
     assert abs(metadata['length_seconds'] - 3.9) < 0.1
     assert metadata['mime'] == 'audio/mp3'
     assert metadata['track_total'] == u'10' # MP3s can have a track_total
@@ -153,7 +158,8 @@ def test_mp3_bad_channels():
     metadata = MetadataAnalyzer.analyze(filename, dict())
     check_default_metadata(metadata)
     assert metadata['channels'] == 1
-    assert metadata['bit_rate'] == 64876
+    assert metadata['bit_rate'] < 65000
+    assert metadata['bit_rate'] > 63000
     assert abs(metadata['length_seconds'] - 3.9) < 0.1
     assert metadata['mime'] == 'audio/mp3' # Not unicode because MIMEs aren't.
     assert metadata['track_total'] == u'10' # MP3s can have a track_total

